docs/information/supported_adapters.md

```diff
@@ -29,6 +29,22 @@ Zigbee2MQTT officially supports the following adapters:
     <td><a href="https://slae.sh/projects/cc2652/#flashing">Flashing</a></td>
     <td><a href="https://slae.sh/projects/cc2652/">Slae.sh</a></td>
   </tr>
+  <tr>
+    <td><img src="../images/Tubes_USB_Coordinator_CC2652P2_variant.png"></td>
+    <td>Tube's CC2652P2 USB Coordinator</td>
+    <td>Open source hardware CC2652P based USB connected adapter with external antenna and USB extension cable</a></td>
+    <td><a href="https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin">Coordinator</a><br/> <a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/router/Z-Stack_3.x.0/bin/">Router</a></td>
+    <td><a href="https://github.com/tube0013/tube_gateways">Flashing</a></td>
+    <td><a href="https://www.tubeszb.com/">Tube's ZB Store</a></td>
+  </tr>
+  <tr>
+    <td><img src="../images/Tubes_Zigbee_Gateways_CC2652P2_variant.jpeg"></td>
+    <td>Tube's Zigbee Gateways (CC2652P2 variant)</td>
+    <td>Open source hardware CC2652P and ESP32 based Zigbee to Ethernet Coordinator</a></td>
+    <td><a href="https://github.com/tube0013/tube_gateways/tree/main/tube_zb_gw_cc2652p2">Coordinator</a></td>
```

Comment:
Does this device not work with the firmwares from Z-Stack-firmware?

Reply:
It uses the z-stack fw. It's based on the rf-star module

Reply:
Changed to URL to link to Koenkk Z-Stack-firmware and point directly to the file